packages/fetchai/connections/p2p_libp2p_client/connection.py

```diff
@@ -256,6 +341,42 @@ def __init__(self, **kwargs: Any) -> None:
         self._process_messages_task = None  # type: Optional[asyncio.Future]
         self._node_client: Optional[NodeClient] = None
 
+        self._send_queue: Optional[asyncio.Queue] = None
```

Comment:
(above) `_reader` and `_writer` not used

Reply:
done